packages/caver-core-helpers/src/validateFunction.js

```diff
@@ -567,7 +572,7 @@ function checkDeployEssential(transaction) {
         return new Error('"data" is missing')
     }
 
-    if (transaction.to !== undefined && transaction.to !== '0x') {
+    if (transaction.to !== undefined && transaction.to !== '0x' && transaction.to !== '0x0000000000000000000000000000000000000000') {
```

Comment:
Do we need to add check condition about zero address in SmartContractDeploy? 
According to Klaytn Docs, the 'to' field in SmartContractDeploy must has nil.

Reply:
Oh, i think this logic doesn't need! So i removed this. Thank you :)